### ISSUE TITLE
The share and block/report sub menus won't open on Chrome

### ISSUE THREAD

**Issue by Spabobin** [Bl4Cc4t/GoodTwitter2#196]
In the last day or so I've had this bug on Chrome only, Firefox is unaffected. Trying to click either of the drop down menus on a tweet (the arrow on the top right or the "share" options on the bottom right) won't work. The top right one opens a blank box and the bottom one just takes me to the top of the page.

![twitterbug](https://user-images.githubusercontent.com/66586672/92987719-c0786000-f48a-11ea-852f-bc82a4a75dee.png)

Chrome Version 85.0.4183.102 (Official Build) (64-bit)
Tampermonkey v4.10

edit: I was using my laptop when I submitted this and had uBlock disabled; I tested on my desktop PC and the bug isn't happening on there with the exact same setup, so idk why this would only be affecting my laptop. Might not be related to GoodTwitter2 unless other people have had this problem too.

**Comment by Bl4Cc4t** [comment]
do you perhaps have any other extensions installed on your laptop?

**Comment by Spabobin** [comment]
I figured it out. For some reason, simply turning off uBlock by clicking the disable button didn't stop it from running my filters. When I removed the filters entirely then it started working again. It was the filter I had set up to get rid of this annoying box that wants me to change tweets to "top tweets first", that shows up every single time I load my timeline. 

![jkjlkjkljkljlkjkljl](https://user-images.githubusercontent.com/66586672/92994693-19171f80-f4c2-11ea-93c4-e07d73f6721b.png)
